main.go

```diff
@@ -144,6 +109,49 @@ func createOrUpdateComment(ctx context.Context, title, details string) {
 	}
 }
 
+func buildCommentBody(commentMarker, summary, reportTable string) string {
```

Comment:
Broke this out into a function, allow for an easy test as well (since it's getting more complex now with the `<detail>` block).